linera-service/tests/end_to_end_tests.rs

```diff
@@ -85,6 +85,38 @@ impl FungibleApp {
     }
 }
 
+struct NativeFungibleApp(FungibleApp);
```

Comment:
So we can't because the balance query for the native fungible will be different. For the `FungibleApp` it just queries the existing state, we don't have state on the native fungible app

Reply:
But is that difference observable? Don't you make the same query, and get the same result?

Reply:
The query for fungible is `accounts {{ entry(key: {}) {{ value }} }}`, because it queries the state map directly. For native fungible the query is `balance(owner: {})`, so it's different queries :/

Reply:
Ah, I see!
But we _could_ make the queries look identical, I think? And maybe we should?
We could allow both kinds of queries on both applications.

Reply:
True, I could do that. Let me look into it